commit db90e16d17acc778efdfff6dca69767b47b60cbd
Author: Benda Xu <heroxbd@gentoo.org>
Date:   Sat Aug 19 17:31:38 2017 +0900

    d/s/examples/N/Importing Notebooks.ipynb: disable execution.
    
      This notebook example includes writing to the directory of another
      module, usually resulting in a doc build failure.  Therefore we
      disable its execution by nbsphinx.
    
    Closes: #2372
    See-Also: https://bugs.gentoo.org/show_bug.cgi?id=626860

diff --git a/docs/source/examples/Notebook/Importing Notebooks.ipynb b/docs/source/examples/Notebook/Importing Notebooks.ipynb
index ee42ad526..d6db5395e 100644
--- a/docs/source/examples/Notebook/Importing Notebooks.ipynb	
+++ b/docs/source/examples/Notebook/Importing Notebooks.ipynb	
@@ -511,6 +511,9 @@
  ],
  "metadata": {
   "gist_id": "6011986",
+  "nbsphinx": {
+   "execute": "never"
+  },
   "kernelspec": {
    "display_name": "Python 3",
    "language": "python",
